core/lib/include/xfa.h

```diff
@@ -42,16 +42,18 @@ _Pragma("GCC diagnostic ignored \"-Warray-bounds\"")
  *
  * @internal
  */
-#define _XFA(name, prio) __attribute__((used, section(".xfa." #name "." #prio)))
+#define _XFA(name, prio) \
```

Comment:
You have to also

    #include "compiler_hints.h"

here now :wink:

Reply:
Ah regular C then - I somehow managed to find two fragile examples and concluded that the compiler hints might be pulled in with some buildsystem blackmagic  🫣